docs/guides/admin/docs/configuration/security.jwt.md

```diff
@@ -76,8 +75,16 @@ sections found in `etc/security/mh_default_org.xml`. Some of the options are con
   <property name="loginHandler" ref="jwtLoginHandler" />
   <!-- Throws an exception if a request is missing the configured header (default: true) -->
   <property name="exceptionIfHeaderMissing" value="false" />
-  <!-- Logs the header contents for debug purposes (default: false) -->
-  <property name="debug" value="false" />
```

Comment:
I just realized that the old filter had the same property, and as this PR goes into 12, it should not contain breaking changes. If you want that both filters behave similarly, revert the changes, or else do not remove the property from the existing filter.

Reply:
Good point. I will extract the change into a new PR. That has to target `develop` then, right? 🤔

Reply:
:+1:

Reply:
Done: https://github.com/opencast/opencast/pull/4511